python/ray/data/dataset.py

```diff
@@ -2231,6 +2231,8 @@ def iter_batches(
         batch_size: Optional[int] = None,
         batch_format: str = "native",
         drop_last: bool = False,
+        random_block_order: bool = False,
```

Comment:
Could we actually make this a transformation on the dataset, like `ds.randomize_block_order()`? This will make it possible to support this generally as a dataset_config in Train, instead of requiring users to always add this in their iterator args / to_tf / to_torch.

Reply:
+1

Reply:
Oh, yeah this is a great idea! Will update PR accordingly.